src/main/kotlin/me/apella/chat/extension/Utils.kt

```
@@ -0,0 +1,14 @@
+package me.apella.chat.extension
+
+import java.util.UUID
+
+fun Any.toUUID(): UUID? {
+    return when(this) {
+        is String -> try {
+            UUID.fromString(this)
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)